package-lock.json

```diff
@@ -4047,7 +4110,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
I've got a difference when I `npm i`. 
It seems we could remove this line (`"optional": true`) as well as few others below.
@etpinard please let me know if you wanted me to push a commit to remove those?

Reply:
Which version of `npm` are you using? I'm on 6.7.0

Reply:
My npm version is at 6.4.1

Reply:
Can you bump your npm version to 6.7.0 and try `npm i` again?

Reply:
Cool! With 6.7.0 I've got no difference.